New page for public workspaces (#2996)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/public-workspaces.md

---
title: "Public workspaces"
order: 76
page_id: "public_workspaces"
warning: false
contextual_links:
  - type: section
    name: "Additional Resources"
  - type: subtitle
    name: "Related Blog Posts"
  - type: link
    name: "Creating workspaces"
    url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"

---

> [Public workspaces](https://blog.postman.com/public-workspaces-why-we-created-them-what-you-can-do/) enable you to browse and collaborate on entities with anyone across the world. Before you create a public workspace, navigate to team profile settings and enable your public team profile. This will ensure your team's profile will show up on the Public API Network.

Reviewer: I'd remove the "browse and" in the first sentence to make it "enable you to collaborate on entities" as I think it makes it a stronger statement

> [Public workspaces](https://blog.postman.com/public-workspaces-why-we-created-them-what-you-can-do/) enable you to browse and collaborate on entities with anyone across the world. Before you create a public workspace, navigate to team profile settings and enable your public team profile. This will ensure your team's profile will show up on the Public API Network.

Reviewer: "navigate to team profile settings" -> "navigate to your [team profile settings](https://go.postman.co/settings/team/general)"

> <img alt="Enable team profile" src="https://assets.postman.com/postman-docs/enable-team-profile.jpg" height="500px"/>

Reviewer: Not a priority but I would crop this further on all sides to make the text clearer to read.

## Contents

* [Creating a public workspace](#creating-a-public-workspace)
* [Accessing a public workspace](#accessing-a-public-workspace)
* [Sharing a public workspace](#sharing-a-public-workspace)
* [Joining a public workspace](#joining-a-public-workspace)
* [Leaving a public workspace](#leaving-a-public-workspace)
* [Adding elements to a public workspace](#adding-elements-to-a-public-workspace)
* [Removing elements from a public workspace](#removing-elements-from-a-public-workspace)
* [Deleting a public workspace](#deleting-a-public-workspace)
* [Managing public workspace members](#managing-public-workspace-members)
* [Next steps](#next-steps)

## Creating a public workspace

> There are four types of workspaces - personal, team, private and public workspaces. When you open a workspace, Postman will provide an overview of its contents, activity, and members in a new tab.

Reviewer: Add comma after "private"

Open the workspace menu, then click **+ New Workspace**.

<img alt="New workspace" src="https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg" width="400px"/>

> Specify a workspace name and summary. For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).

Reviewer: I might add a blank line after the first sentence to separate it out (may help non-free-or-team users to scroll down the page a bit)

> Specify a workspace name and summary. For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).

Reviewer: "For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles)."

I'd rephrase a bit to something like the following if it sounds good

"If you are on a Postman __Team__ or __Free__ plan, you can create a public workspace by changing the workspace's visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles)."

[![Public workspace](https://assets.postman.com/postman-docs/create-public-workspace-v8.jpg)](https://assets.postman.com/postman-docs/create-public-workspace-v8.jpg)

> You can also create a public workspace by selecting the team workspace and changing the team's visibility dropdown to __Public__ and clicking __Request to Change Visibility__.

Reviewer: I would adjust to something like

"You can also create a public workspace by selecting the team workspace, changing the team's visibility dropdown to __Public__, then clicking __Request to Change Visibility__."

> [![Overview team workspace](https://assets.postman.com/postman-docs/overview-team-workspace.jpg)](https://assets.postman.com/postman-docs/overview-team-workspace.jpg)

Reviewer: I'm not sure where to look for the above in this screenshot. Could you make it more obvious by cropping to where the action is and/or selecting?

Author: I will remove this screenshot and keep the one after Business and Enterprise users (zoomed-in bit).

> For __Business__ and __Enterprise__ users, you need a community manager's approval to change the workspace visibility to __Public__.

Reviewer: I would rephrase this similar to the sentence for free/team above

<img alt="Visibility" src="https://assets.postman.com/postman-docs/visibility-options-v8.jpg" width="400px"/>

> Once you make a workspace public, the notification is sent to workspace members. Click the notification bell in the top right corner to view notifications.

Reviewer: ```suggestion
Once you make a workspace public, a notification is sent to workspace members. Click the notification bell in the top right corner to view notifications.
```

<img alt="In app notification" src="https://assets.postman.com/postman-docs/visibility-app-notification.jpg" width="500px"/>

> In addition to this, you will receive an email with the workspace information regarding who has made the change and which workspace was affected with the sharing link for the workspace.

Reviewer: Can we specify who "you" is here—all users or team admins, etc?

> In addition to this, you will receive an email with the workspace information regarding who has made the change and which workspace was affected with the sharing link for the workspace.

Reviewer: Maybe break off " with the sharing link for the workspace." to a separate sentence, like 

". The email will also provide a direct link to the workspace."

<img alt="Email notification" src="https://assets.postman.com/postman-docs/email-notification-public-workspace-v2.jpg" width="500px"/>

> Once you make a workspace public, all entities within that workspace become publicly accessible through the workspace. You need to be a [__workspace admin__](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace to public. Within a team, you cannot have two public workspaces with the same name.

Reviewer: I would put the workspace admin information higher up in this section. The two can't have the same name bit I would change into a note and keep down here.

> You can convert a private workspace directly to public.

## Accessing a public workspace

You can access public workspaces via the [**Workspaces** category in the API Network](https://www.postman.com/explore/workspaces). You can also access personal, team, and private workspaces using the workspace selector at the top of Postman.

[![Access Public Workspaces](https://assets.postman.com/postman-docs/accessing-public-workspaces.gif)](https://assets.postman.com/postman-docs/accessing-public-workspaces.gif)

To view and edit your public workspace details (e.g. name, description), navigate to the workspace selector and hover over a workspace. Then, click **...** and select **View**.

<img alt="Create public workspace dropdown" src="https://assets.postman.com/postman-docs/create-public-workspace-dropdown.jpg" height="350px"/>

## Sharing a public workspace

> You can share a public workspace by sharing its URL directly. To access the public profile URL, navigate to **Your Team**, then click **View Team Settings**.

Reviewer: Can you explain where to get the URL for the workspace specifically after the first sentence?

<img alt="Enable team profile" src="https://assets.postman.com/postman-docs/enable-team-profile.jpg" height="500px"/>

## Joining a public workspace

If you are part of a team in Postman, you can find public workspaces to join. Open the workspaces drop-down from the control at the top of Postman in the left. Search for a public workspace and select the workspace you want to join.

<img alt="Joining a public workspace overview" src="https://assets.postman.com/postman-docs/join-public-workspace-from-selector.jpg" height="350px"/>

Select the public workspace, then click **Join Workspace** at the top right.

<img alt="Join public workspace" src="https://assets.postman.com/postman-docs/join-public-workspace-from-overview-v8.jpg"/>

> Once you join the workspace, you can start creating collections, sending requests, visualizing your responses and even much more.

Reviewer: ```suggestion
Once you join the workspace, you can start creating collections, sending requests, visualizing your responses, and more.
```

## Leaving a public workspace

> To leave a public workspace, select the **...** on the right corner of the workspace overview and **Leave workspace**. Click **Leave** to finalize leaving the workspace.

Reviewer: ```suggestion
To leave a public workspace, select the **...** on the right corner of the workspace overview > **Leave workspace**. Click **Leave** to finalize leaving the workspace.
```

<img alt="Adding an environment to public workspace" src="https://assets.postman.com/postman-docs/share-environment-public-workspace.jpg" height="400px"/>

Select the public workspace you want to share to, and choose whether you also want to remove the element from its current workspace. Click **Share and Continue**.

<img alt="Share element to public workspace" src="https://assets.postman.com/postman-docs/share-element-to-public-workspace.jpg" height="400px"/>

## Removing elements from a public workspace

When you remove an element from a public workspace, it is no longer visible in that particular workspace. The element is still available in any other workspace where it has been added.

From Postman, you can remove a collection from the sidebar. Click **...** to open the collection menu. Select **Remove from workspace** and confirm that you would like to remove the collection from the current workspace. The collection will no longer be visible in the workspace.

<img alt="Removing a collection to public workspace" src="https://assets.postman.com/postman-docs/remove-collection-public-workspace.jpg" height="500px"/>

> Also, you can remove an environment from the sidebar. Click **...** next to the environment menu. You can also click **...** next to the Share button in the Environment tab. Select **Remove from workspace** and confirm that you would like to remove the environment from the current workspace. The environment will no longer be visible in the workspace.

Reviewer: ```suggestion
To remove an environment from the sidebar, click **...** next to the environment menu or click **...** next to the Share button in the Environment tab. Select **Remove from workspace** and confirm that you would like to remove the environment from the current workspace. The environment will no longer be visible in the workspace.
```

<img alt="Removing an environment to public workspace" src="https://assets.postman.com/postman-docs/remove-environment-public-workspace.jpg"/>

> Note that deleting an element is different to removing it. When you delete an element it is no longer available in any workspace. When you remove an element from a specific workspace, it will still be available in any other workspaces it was in.

## Deleting a public workspace

> Public workspaces must have their visibility reverted in order to be deleted. If you try to delete a public workspace by clicking **...** on the right corner of the workspace overview, the option to delete will be greyed out.

Reviewer: Instead of explaining the greyed out option, I'd give a quick review of how to change visibility, then delete, in this section

<img alt="Deleting a public workspace" src="https://assets.postman.com/postman-docs/delete-option-greyed-public-workspace.jpg"/>

## Managing public workspace members

> If you're a team administrator you can manage the members of your public workspace, via the **Members** tab. You have the permission to add and remove any member from a public workspace. To add members to a public workspace, navigate to the Members and groups section, and click **Invite**.

Reviewer: ```suggestion
If you're a team administrator you can add or remove any member from your public workspace via the **Members** tab. To add members to a public workspace, click **Invite**.
```

<img alt="add members for public workspace (1)" src="https://assets.postman.com/postman-docs/remove-team-member-workspace-overview.jpg" width="400px"/>

Give a name, email or group name, click **Add** to add a team member to the workspace, then **Send Invites**.

<img alt="add workspace members public workspace (3)" src="https://assets.postman.com/postman-docs/add-member-to-public-workspace.jpg"/>

> Another way to add workspace members is to click **Manage Team** and select **Invite Users**.

Reviewer: If not as important to have here, you could consider linking out to `/docs/administration/managing-your-team/managing-your-team/#inviting-to-a-team` and removing the steps here

<img alt="add workspace members public workspace (2)" src="https://assets.postman.com/postman-docs/manage-team-v8.jpg"/>

Enter an email address, click **Add** to the team, select the roles you would like to assign to them, then select **Send Invitations**. You can also generate a link and invite people to your team by sharing the link with them.

<img alt="Edit members for a public workspace" src="https://assets.postman.com/postman-docs/edit-member-public-workspace.jpg"/>

> Team members will receive an email and in-app notification when added to a public workspace.

> To remove any team member, click the trash icon near the person you want to remove, and select **Remove User From Team**.

Reviewer: ```suggestion
To remove any team member, click the trash icon next to the individual you want to remove from the workspace, and select **Remove User From Team**.
```

<img alt="add workspace members public workspace (2)" src="https://assets.postman.com/postman-docs/manage-team-v8.jpg"/>

Enter an email address, click **Add** to the team, select the roles you would like to assign to them, then select **Send Invitations**. You can also generate a link and invite people to your team by sharing the link with them.

<img alt="Edit members for a public workspace" src="https://assets.postman.com/postman-docs/edit-member-public-workspace.jpg"/>

> Team members will receive an email and in-app notification when added to a public workspace.

To remove any team member, click the trash icon next to the individual you want to remove from the workspace, and select **Remove User From Team**.

<img alt="Remove team members for a public workspace" src="https://assets.postman.com/postman-docs/remove-member-public-workspace.jpg"/>

Click X next to the team member you want to remove from the public workspace, and click **Save**.

<img alt="Remove members for a public workspace from overview" src="https://assets.postman.com/postman-docs/remove-team-member-workspace-overview.jpg" width="400px"/>

> ## Next steps

Reviewer 2: @sowmyaraj92 we could also add a section about adding categories to a public workspace from the profile page.
